commands/util/binary.js

```diff
@@ -0,0 +1,152 @@
+const { SlashCommandBuilder, EmbedBuilder } = require("discord.js");
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName("binary")
+    .setDescription("Binary and Decimal Conversions")
+    .addSubcommand(subcommand =>
+      subcommand
+        .setName("encode")
+        .setDescription("Convert a number to binary")
+        .addIntegerOption(option =>
+          option
+            .setName("number")
+            .setDescription("Pick a number to convert into binary")
+            .setRequired(true)
+        )
+    )
+    .addSubcommand(subcommand =>
+        subcommand
+          .setName("decode")
+          .setDescription("Convert a binary number to a decimal number")
+          .addIntegerOption(option =>
+            option
+              .setName("binary")
+              .setDescription("Pick a binary number to convert into decimals")
+              .setRequired(true)
+        )
+    )
+    .addSubcommand(subcommand =>
```

Comment:
A remark in general:
Program your arithmetic functions in such a way that it doesn't matter what the user puts in, the bot does the arithmetic. 

If the user put in a regular integer, it converts it to binary and does the math. If the user puts in a binary number, it keeps it as a binary number and does the arithmetic

Reply:
bro you can't do that 10 is a decimal and a binary number